registry/client.go

```diff
@@ -62,6 +65,31 @@ type credentials struct {
 	password string
 }
 
+type schemaInfoPayload struct {
+	Schema  string `json:"schema"`
+	ID      int    `json:"id"`
+	Version int    `json:"version"`
+	Subject string `json:"subject"`
```

Comment:
We dont need to spend resources unmarshalling this, it was passed into the function.